sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/InvitationService.java

```diff
@@ -150,27 +153,34 @@ public Invitation retryInvitation(Long principalId, String email) {
         invitationFromDb.setCreatedBy(userService.getUserById(principalId));
         invitationFromDb = updateInvitation(invitationFromDb);
         sendEmail(invitationFromDb);
+
+        insertInvitationUrl(invitationFromDb);
```

Comment:
Can `invitationFromDb` be renamed to `invitation`? I can't see another variable with conflicting name in scope of this method, so i assume the answer is yes